### ISSUE TITLE
Meta-Sim: Learning to Generate Synthetic Datasets 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#61]
# TL;DR

They propose Meta-Sim that generates a large dataset of labeled data for applications such as self-driving by generating the corresponding image from a document (label) of graph structure. The optimization is achieved not only by the loss of measuring whether the generated images correspond to the document, but also by the loss of optimizing the task (e.g., semantic segmentation).
![Figure 12  (left) samples from our prob  grammar, (middle) Meta-Sim's corresponding samples, (right) random samples from KITTI](https://user-images.githubusercontent.com/24524018/91837889-9acba980-ec87-11ea-93d2-41d23fad85d8.png)
![Meta-Sim](https://user-images.githubusercontent.com/24524018/91837896-9dc69a00-ec87-11ea-9720-2a3095d4288a.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1904.11621

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Amlan Kar, Aayush Prakash, Ming-Yu Liu, Eric Cameracci, Justin Yuan, Matt Rusiniak, David Acuna, Antonio Torralba, Sanja Fidler
- NVIDIA
- University of Toronto 
- Vector Institute
- MIT

# Methods

# Results

# Comments
